src/fileUtils.ts

```diff
@@ -0,0 +1,15 @@
+import * as vscode from 'vscode';
+
+export async function getFilePath(): Promise<vscode.Uri | undefined> {
+    const fileUri = await vscode.window.showSaveDialog({
+        defaultUri: vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri : undefined,
+        filters: {
+            'xml': ['xml'],
```

Comment:
New feature: Can we add a nice name based on the node selected?

Reply:
Are you referring to `saveLabel` and `title` fields of [`SaveDialogOptions`](https://code.visualstudio.com/api/references/vscode-api#SaveDialogOptions)?

Reply:
No, the default file name suggested on save is `untitled.xml`. Is it possible for there to be a better default?

Reply:
Thanks @thegecko

@QuocTrung76 that means `defaultUri` needs to be updated in SaveDialogOptions (I leave it up to you whether saveLabel or title also should be updated)

Reply:
Hi @thegecko , I set the default filename to 'registers.xml', is it fine or do you have any recommendations?
Hi @jonahgraham , the saveDialogOptions do not allow setting the default filename by options, I have modified a bit to put it.

Reply:
`registers.xml` is better IMO. I was wondering if the name could be based on the node name you are exporting?

Reply:
I'm happy to leave this as just `registers.xml` for now 👍